cpp/include/cudf/io/detail/parquet.hpp

```diff
@@ -113,11 +113,11 @@ class writer {
    */
   std::unique_ptr<std::vector<uint8_t>> write(
     table_view const& table,
-    const table_metadata* metadata            = nullptr,
-    bool return_filemetadata                  = false,
-    const std::string column_chunks_file_path = "",
-    bool int96_timestamps                     = false,
-    rmm::cuda_stream_view stream              = rmm::cuda_stream_default);
+    const table_metadata* metadata                = nullptr,
+    bool return_filemetadata                      = false,
+    const std::string column_chunks_file_path     = "",
+    std::vector<uint8_t> const& decimal_precision = {},
```

Comment:
Just noticed that these parameters don't need default values. Since this is not the only API where this needs to be fixed, it should be done in a separate PR. Commenting here just as a note to follow up on later.

Reply:
Added issue https://github.com/rapidsai/cudf/issues/7068 so we wouldn't forget about this.